splink/comparison_library.py

```diff
@@ -28,6 +29,7 @@
     "JaroAtThresholds",
     "JaroWinklerAtThresholds",
     "DistanceFunctionAtThresholds",
+    "PairwiseStringDistanceFunctionAtThresholds",
```

Comment:
Needs the same in splink/comparison_level_library.py

Reply:
Right you are!